Controls/MotorController.h

```diff
@@ -0,0 +1 @@
+void CANWriter (int CAN_ID,int CAN_MESSAGE);
```

Comment:
Good, but in C especially with embedded systems we want to minimize the memory use. So it would be better to specify size for each of the parameters as the minimum size that will meet the requirements. For example, the ID can be a uint16_t since the ID will only ever be 11 bits, so 16 bits is enough to contain it.